mappings/net/minecraft/world/EntityView.mapping

```diff
@@ -21,30 +21,67 @@ CLASS net/minecraft/class_1924 net/minecraft/world/EntityView
 		ARG 1 entity
 		ARG 2 maxDistance
 	METHOD method_18461 getClosestPlayer (Lnet/minecraft/class_4051;DDD)Lnet/minecraft/class_1657;
+		ARG 1 targetPredicate
 		ARG 2 x
 		ARG 4 y
 		ARG 6 z
 	METHOD method_18462 getClosestPlayer (Lnet/minecraft/class_4051;Lnet/minecraft/class_1309;)Lnet/minecraft/class_1657;
+		ARG 1 targetPredicate
+		ARG 2 entity
 	METHOD method_18463 getClosestPlayer (Lnet/minecraft/class_4051;Lnet/minecraft/class_1309;DDD)Lnet/minecraft/class_1657;
+		ARG 1 targetPredicate
+		ARG 2 entity
 		ARG 3 x
 		ARG 5 y
 		ARG 7 z
-	METHOD method_18464 getPlayersInBox (Lnet/minecraft/class_4051;Lnet/minecraft/class_1309;Lnet/minecraft/class_238;)Ljava/util/List;
+	METHOD method_18464 getPlayers (Lnet/minecraft/class_4051;Lnet/minecraft/class_1309;Lnet/minecraft/class_238;)Ljava/util/List;
+		ARG 1 targetPredicate
+		ARG 2 entity
+		ARG 3 box
 	METHOD method_18466 getTargets (Ljava/lang/Class;Lnet/minecraft/class_4051;Lnet/minecraft/class_1309;Lnet/minecraft/class_238;)Ljava/util/List;
 		ARG 1 entityClass
 		ARG 2 targetPredicate
 		ARG 3 targettingEntity
 		ARG 4 box
-	METHOD method_18467 getEntities (Ljava/lang/Class;Lnet/minecraft/class_238;)Ljava/util/List;
+	METHOD method_18467 getEntitiesExcludingSpectators (Ljava/lang/Class;Lnet/minecraft/class_238;)Ljava/util/List;
```

Comment:
```suggestion
	METHOD method_18467 getNonSpectatingEntities (Ljava/lang/Class;Lnet/minecraft/class_238;)Ljava/util/List;
```
Might need a better term for "actually interacting/playing in the game"

Reply:
> actually interacting/playing in the game

This excludes player in the "spectator" (command argument name!) game mode.